core/components/formalicious/model/formalicious/snippets/formalicioussnippetrenderform.class.php

```diff
@@ -226,21 +248,23 @@ public function run(array $properties = [])
                 $parameters['formaliciousTplNavigationItem'] = $this->getProperty('tplNavigationItem');
                 $parameters['formaliciousTplNavigationWrapper'] = $this->getProperty('tplNavigationWrapper');
 
-                if ($currentStep <= 1) {
```

Comment:
Why do you think we need to remove `prevUrl` if current step less or equal `1`? There will be problem if somebody use this placeholder and it will disappear in new version.

Can you please describe this or return `prevUrl` for first step?

Reply:
I may be not accounting for a particular scenario, but why would you show a previous URL if you're on the first step?

Reply:
I agree with you. But if we delete this, we should change major version number I think. Because this broke backward compatibility. But maybe I'm wrong. I will ask somebody

Reply:
Ok, let me know. FYI, I verified that this change we're debating does not affect the functionality I'm introducing, so it can be reverted back if needed. If I remember, I made that change because the original line in the code caught my eye and didn't make sense to me.

Reply:
@smg6511 First of all, thank you for putting in the work!

> I verified that this change we're debating does not affect the functionality I'm introducing, so it can be reverted back if needed. 
Can you revert this part please, so we don't do any breaking changes?

Also could you add a section in the readme describing what you added and how that can be implemented? This will make it easier for other users to find out that it is possible and how they can achieve this.